notebook/static/tree/js/notebooklist.js

```diff
@@ -595,6 +597,20 @@ define([
             $('.delete-button').css('display', 'none');
         }
 
+        // View is visible when an item is renderable or downloadable
```

Comment:
I considered that, but I have already found this useful (eg, for "grading" in a non-nbgrader environment).